src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -49,86 +60,112 @@ def llm_eval_binary(
     return [_snap_to_rail(response, rails_set) for response in responses]
 
 
+"""
+
+Args:
+    dataframe (pd.DataFrame):
+
+    rails_map: The mapping to use to return the values. Using a dict here
+    but it can take the OrderedDict of RAILS_MAP from library
+
+Returns:
+    List[List[str]]:
+"""
+
+
 def run_relevance_eval(
     dataframe: pd.DataFrame,
     model: BaseEvalModel,
     template: Union[PromptTemplate, str] = RAG_RELEVANCY_PROMPT_TEMPLATE_STR,
-    query_column_name: str = "attributes.input.value",
-    retrieved_documents_column_name: str = "attributes.retrieval.documents",
-) -> List[List[Optional[bool]]]:
-    """Given a pandas dataframe containing queries and retrieved documents,
-       classifies the relevance of each retrieved document to the corresponding
-       query using an LLM.
+    rails: List[str] = list(RAG_RELEVANCY_PROMPT_RAILS_MAP.values()),
+    query_column_name: str = OPENINFERENCE_QUERY_COLUMN_NAME,
+    document_column_name: str = OPENINFERENCE_DOCUMENT_COLUMN_NAME,
```

Comment:
having the OpenInference column name be the default might just cause a huge impedance mismatch here - I think it might make more sense to make evals work "out of the box" on standard dataframes and then have a remapping strategy.

For now I don't mind the "parameterization" of the columns but I think we can do a step better where you don't have to do this - like we can just detect if the openInference columns exist and do the re-mapping for them.

Did you consider this? I think it might be better as it would keep the call signature significantly simpler